zipkin-finagle-kafka/src/main/java/zipkin/finagle/InternalJavaDocDummy.java

```diff
@@ -0,0 +1,10 @@
+package zipkin.finagle;
+
+/**
+ * This file is here to allow maven central deploys and avoid the complexity of trying to convert
+ * scala to javadoc.
+ *
+ * See https://issues.sonatype.org/browse/OSSRH-2932
+ */
+public class InternalJavaDocDummy {
```

Comment:
timeboxed an hour trying to get this to work, but various compile/config fails led me to just use the normal work-around https://github.com/typesafehub/genjavadoc